components/amorphic-di-contracts/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {},
+  "description": "Typescript contracts for amorphic dependency injection",
+  "name": "amorphic-di-contracts",
```

Comment:
Can we make this an @havenlife-public/framework-types? Amorphic di contracts seems like a mouthful.

Reply:
done!